src/components/inputCheckbox/AposCheckbox.vue

```diff
@@ -64,18 +70,26 @@ export default {
     };
   },
   computed: {
-    isChecked: function () {
-      return this.value.includes(this.choice.value);
+    // Handle the local check state within this component.
+    checkProxy: {
```

Comment:
This is a flow I got from [a discussion about `v-model` on custom components](https://github.com/vuejs/vue/issues/3838#issuecomment-642242127). This is useful to be able to high-jack the event emission when the checkbox is indeterminate.